.circleci/config.yml

```diff
@@ -99,6 +99,13 @@ jobs:
       - *install_theme
       - *update_version
       - *save_npm_cache
+      # Set PUBLIC_URL based on the bucket prefix. PUBLIC_URL is equivalent to CRA's package.json homepage setting https://create-react-app.dev/docs/advanced-configuration/
+      - run:
+          name: "Set PUBLIC_URL for review branch"
+          environment:
+            NODE_PATH: /usr/local/lib/node_modules
+          # Two env vars as PUBLIC_URL seems to be blank when running jest even if we set it.
+          command: URL=$(npm run --silent public-url); echo "export PUBLIC_URL=$URL && export E2E_PUBLIC_URL=$URL" >> $BASH_ENV
```

Comment:
`$BASH_ENV` is an obscure CircleCI feature. It's set to the path of a file that's sourced as part of the initialisation of the shell used for each step so appending env variable exports to it is a way of ensuring they're available in subsequent steps.